app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,21 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(
+            self,
+            nickname: str,
+            favourite_dish: str,
+            skill_level: int
+    ) -> None:
+        super().__init__(nickname, favourite_dish)
+        self.__skill_level = skill_level
```

Comment:
![image](https://github.com/mate-academy/py-elves-and-dwarves/assets/103201010/d5fa00f1-ca97-4d96-afd0-a984dc7a8fbe)
It seems to me, that this is not protected attr )
fix this everywhere